ci/requirements-demos.txt

```
@@ -154,6 +154,7 @@ xmltodict==0.12.0
     # via motmetrics
 zipp==3.5.0
     # via importlib-metadata
+inflect>=5.3.0
```

it is preferred to pin specific version on testing environment
```suggestion
inflect==5.3.0
```